todoapp.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=My To Do App
+
+[Service]
+User=ec2-user
+WorkingDirectory=/opt/todoapp
+ExecStart=/home/ec2-user/.local/bin/poetry run flask run --host=0.0.0.0 --port=2022
```

Comment:
Port 2022 threw me - thought I was failing to connect but I just wasn't looking for the right port! That's on me, but is a good reminder of why we should be documenting this, both how it can be run (i.e. the docs you have added) as well as what it's achieving (deploying to <http://18.135.202.168:2022/>)

Reply:
Hah, sorry about that! I tried a different port as I was troubleshooting some issues with the systemd task.